map/games/another_world.xml

```diff
@@ -59285,6 +59285,9 @@ Update to 3.7.2  Fixes "Bunker" units throwing error when "aARadar" tech is acti
 <br>
 <br>
 Update to 3.7.3  Fixes missing "city0" unit images.
+<br>
+<br>
+Update to 3.7.  Fixes missing "city0_hit_damaged" unit images. Also fixes "helicopter_r_lr_jp".
```

Comment:
Look to be missing the '4' on 3.7.4

Reply:
oops I guess fix it next time something comes up since this is already merged ?

Reply:
IMO just submit a follow-up fix while we're aware of this.